integration_tests/suite/test_subscriptions.py

```diff
@@ -1,28 +1,37 @@
 # Copyright 2017-2019 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from hamcrest import assert_that
-from hamcrest import calling
-from hamcrest import contains
-from hamcrest import empty
-from hamcrest import has_entries
-from hamcrest import has_entry
-from hamcrest import has_key
-from hamcrest import has_item
-from hamcrest import has_property
-from hamcrest import not_
+from hamcrest import (
+    assert_that,
+    calling,
+    contains,
+    empty,
+    equal_to,
+    has_entries,
+    has_entry,
+    has_key,
+    has_item,
+    has_property,
+    not_,
+)
 from wazo_webhookd_client.exceptions import WebhookdError
-from xivo_test_helpers.auth import MockUserToken
 from xivo_test_helpers.hamcrest.raises import raises
 
-from .helpers.base import BaseIntegrationTest
-from .helpers.base import VALID_TOKEN
+from .helpers.base import (
+    BaseIntegrationTest,
+    MASTER_TOKEN,
+    MASTER_TENANT,
+    WAZO_UUID,
+    USER_1_TOKEN,
+    USER_2_TOKEN,
+    USER_1_UUID,
+    USERS_TENANT,
+    OTHER_USER_TOKEN,
+)
 from .helpers.fixtures import subscription
```

Comment:
[Outside of scope (please do not change it in this PR)]
In the others repo we just `from .helpers import fixtures` and use `fixtures.subscription`. Since the name fixtures is not in the name of the object, is more appropriate. And we do not need to set some weird suffix (i.e. : `subscription_`)

Reply:
Completely agree